ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -72,9 +77,9 @@ const mapStateToProps = (state, ownProps) => {
   const txParams = selectedTransaction?.txParams
     ? selectedTransaction.txParams
     : {
-        gas: send.gasLimit || '0x5208',
-        gasPrice: send.gasPrice || getAveragePriceEstimateInHexWEI(state, true),
-        value: sendToken ? '0x0' : send.amount,
+        gas: gasLimit || '0x5208',
```

Comment:
Nit: can we add comments giving the value for hex values?

Reply:
I know you didn't add this...

Reply:
Can we use the constant here?